config.py

```diff
@@ -268,6 +268,15 @@ def validate_config(CONFIG: CONFIG_DICT_TYPE) -> None:
             config_assert(online_section.get("move_quality") != "suggest" or not online_section.get("enabled"),
                           f"XBoard engines can't be used with `move_quality` set to `suggest` in {subsection}.")
 
+    matchmaking = CONFIG.get("matchmaking") or {}
+    matchmaking_enabled = matchmaking.get("enabled") or False
+    # `, []` is there only for mypy. It isn't used.
+    matchmaking_has_values = (matchmaking.get("challenge_initial_time", [])[0] is not None and 
+                              matchmaking.get("challenge_increment", [])[0] is not None or
+                              matchmaking.get("challenge_days", [])[0] is not None)
+    config_assert(not matchmaking_enabled or matchmaking_has_values,
+                  "The time control to challenge other bots isn't set.")
```

Comment:
I think a more specific message would be helpful: `The time control to challenge other bots is not set. Either lists of challenge_initial_time and challenge_increment is required, or a list of challenge_days, or both.`